sql/catalyst/src/main/resources/julian-gregorian-rebase-micros.json

```diff
@@ -5,14 +5,14 @@
 }, {
   "tz" : "Africa/Accra",
   "switches" : [ -62135769600, -59006448000, -55850688000, -52694928000, -46383408000, -43227648000, -40071888000, -33760368000, -30604608000, -27448848000, -21137328000, -17981568000, -14825808000, -12219292800, -2208988800 ],
-  "diffs" : [ 172852, 86452, 52, -86348, -172748, -259148, -345548, -431948, -518348, -604748, -691148, -777548, -863948, 52, 0 ]
+  "diffs" : [ 173768, 87368, 968, -85432, -171832, -258232, -344632, -431032, -517432, -603832, -690232, -776632, -863032, 968, 0 ]
```

Comment:
Regenerate `julian-gregorian-rebase-micros.json` using `generate 'julian-gregorian-rebase-micros.json'` in `RebaseDateTimeSuite` with Java 8u352